.github/workflows/build.yml

```diff
@@ -273,9 +273,19 @@ jobs:
       - name: check for noisy stdout lines
         id: noisy
         run: |
-          grep --after-context=1 "stdout" stdout.log && ret=0
-          grep --after-context=1 "stderr" stdout.log && new=0
-          if [ "${ret}" == "0" ] || [ "${new}"  == "0" ]; then exit 1; fi
+          buildMsgs="$(
+            ##  we exploit `lake`s replay feature: since the cache is present, running
+            ##  `lake build` will reproduce all the outputs without having to recompute
+            lake build Mathlib Archive Counterexamples |
+            ##  we filter out the output lines that begin with `[xx/yy]`, where xx, yy
+            ##  are either numbers or ?
+            ##  we keep the rest, which are actual outputs of the files
+            awk '!($1 ~ "^[[?0-9]*/[?0-9]*]"){ print $0 }')"
```

Comment:
This is very likely to break in rc2, after Mac is finished with https://github.com/leanprover/lean4/pull/4127. We'll just adapt as needed.

Reply:
Ok, I'm happy to monitor what needs to be changed with rc2!

As is, CI is not really testing for noisy files anymore.